src/playlist-loader.js

```diff
@@ -102,7 +102,7 @@ const updateMaster = function(master, media) {
   return changed ? result : null;
 };
 
-const PlaylistLoader = function(srcUrl, withCredentials) {
+const PlaylistLoader = function(srcUrl, withCredentials, startsPaused) {
```

Comment:
I would rather we adopt the convention that loaders always start paused (a la segment loader) and provide a mechanism to begin loading explicitly. Having the two loader types share a basic lifecycle sounds like a nice simplification to me.